File: SK210 - Java/app/src/main/java/com/example/sk210/util/PriceScanApplication.java
package com.example.sk210.util;

import android.app.Application;
import android.content.Context;

public class PriceScanApplication extends Application {

    private static Context context;

    @Override
    public void onCreate() {
        super.onCreate();
        context = this;
    }

    public static Context getContext() {
        return context;
    }
}

File: SK210 - Kotlin/app/src/main/java/com/exampledv/exemplosk210kot/util/PriceScanApplication.kt
package com.exampledv.exemplosk210kot.util

import android.app.Application
import android.content.Context

class PriceScanApplication : Application() {

    companion object {
        private lateinit var instance: PriceScanApplication

        fun getContext(): Context {
            return instance.applicationContext
        }
    }

    override fun onCreate() {
        super.onCreate()
        instance = this
    }
}
